### ISSUE TITLE
Error using Faster RCNN in MATLAB. 

### ISSUE THREAD

**Issue by faisalalamri** [tensorflow/tensorflow#15241]
Hi there, 
I am working on a project that is about detecting multiple object classes (object classes detection), where the dataset that I am using is MSCOCO (2014). 
The method that I am using is Faster RCNN detection method, where I am using matlab 2017a
Please note that all training is done on a machine with a GPU (NVIDIA Tesla K80)
I have wrote the code attached in the zipped file, dropbox link for downloading is presented below. I have done this code by following the instructions available on matlab transfer learning page; it is about using alexnet as a pre-trained mode for Faster RCNN. Can you kindly have a look at the code and let me know, it has any errors, because I am getting an error when doing the evaluation  
Error is:  
Error using fasterRCNNObjectDetector/parseDetectInputs (line 680)
The size of I must be larger than [227 227]. The minimum size is defined by the network's image input layer.
Error in fasterRCNNObjectDetector/detect (line 447)
            params = this.parseDetectInputs(I, varargin{:});
I have resized the image to [400 400], which passed this errors, but still have another error with plotting the accuracy, IF I use less than 400 (e.g., [300 300]), the error will still appear, which I am not sure why, because alexnet input layer is in the size of [227 227], where 300 is still larger than. 
Error is: I am getting all the average precision (ap), recall values as zeros (0). 

To download the code and all materials: https://www.dropbox.com/sh/9u56ragkuz1ltgd/AADKvUXESezWsoaGfgLyE0Exa?dl=0 
Please note the zipped file contains: 
1) the training (transfer learning) code, named as run_20_classes.m
2) .mat file,  named as pre-trained_20_classes.mat that contains the training and testing labels (annotations), stored as a table, where the first column contains paths and file names to images. The remaining columns contain bounding boxes related to the corresponding images. Each column represents a single object class, such as a person, bicycle, car … etc, as explain on https://uk.mathworks.com/help/vision/ref/trainfasterrcnnobjectdetector.html. This mat file also includes the results of detection, saved as detector  the evaluation results.
3) testing code, named as Testing.m

Training images can be downloaded from: http://images.cocodataset.org/zips/train2014.zip 
Testing images can be downloaded from: http://images.cocodataset.org/zips/val2014.zip 

**Can you please help me on how to improve the accuracy, or if my code is mistaken, can you please correct it?**
Thank you very much for your help and time in advance 

Have I written custom code: I did the code, following the instructions available on matlab both Faster RCNN detection, and transfer learning webpages (https://uk.mathworks.com/help/vision/examples/object-detection-using-faster-r-cnn-deep-learning.html) and (https://uk.mathworks.com/help/nnet/ug/pretrained-convolutional-neural-networks.html#bvm8b5x) respectively
OS Platform and Distribution: Linux, MATLAB2017a
CUDA/cuDNN version: 8.0, V8.0.44
GPU model and memory: NVIDIA Tesla K80
Exact command to reproduce: training code (run_20_classes.m), testing (Testing.m)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
